cms/static/sass/elements/_forms.scss

```diff
@@ -404,7 +404,7 @@ form {
     max-width: 80%;
   }
 
-  .incontext-editor-open-action {
+  .incontext-editor-open-action, .visibility-button {
```

Comment:
Nit: can you use access-button since we are referring to it as "Access Settings" instead of "Visibility Settings"?